pyaedt/hfss.py

```diff
@@ -5834,6 +5834,66 @@ def set_impedance_multiplier(self, multiplier):
         except:
             return False
 
+    @pyaedt_function_handler()
+    def set_phase_center_per_port(self, coordinate_system=None):
+        # type: (list) -> bool
+        """Set phase center per port.
+
+        Parameters
+        ----------
+        coordinate_system : list
+            List of coordinate system per port. The default is ``None``, in which case the
+            default port location is assigned.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
+        References
+        ----------
+
+        >>> oModule.SetPhaseCenterPerPort
+
+        Examples
+        --------
+
+        Set phase center of an antenna with two ports.
+
+        >>> hfss.set_phase_center_per_port(["Global", "Global"])
+
+        """
+
+        if not self.desktop_class.is_grpc_api:  # pragma: no cover
+            self.hfss.logger.warning("Set phase center not working with COM, user must do it manually")
```

Comment:
```suggestion
            self.hfss.logger.warning("Set phase center not working with COM. Set phase center manually")
```
Avoid "user" Use "you" instead wherever possible. In this case, the "you" is implied.